bip-0174.mediawiki

```diff
@@ -483,6 +483,51 @@ The currently defined per-input types are defined as follows:
 | 0, 2
 | [[bip-0371.mediawiki|371]]
 |-
+| MuSig2 Participant Public Keys
+| <tt>PSBT_IN_MUSIG2_PARTICIPANT_PUBKEYS = 0x19</tt>
+| <33 byte plain aggregate pubkey>
+| The MuSig2 aggregate plain public key from the <tt>KeyAgg</tt> algorithm. This key may or may not
+be in the script directly (as x-only). It may instead be a parent public key from which the public keys in the
+script were derived.
+| <tt><33 byte compressed pubkey>*</tt>
+| A list of the compressed public keys of the participants in the MuSig2 aggregate key in the order
+required for aggregation. If sorting was done, then the keys must be in the sorted order.
+|
+|
+| 0, 2
+| [[bip-musig2-psbt.mediawiki|musig2]]
+|-
+| MuSig2 Public Nonce
+| <tt>PSBT_IN_MUSIG2_PUB_NONCE = 0x1a</tt>
+| <33 byte compressed pubkey> <32 byte xonlypubkey> <32 byte hash>
```

Comment:
have you thought about putting `[]` or something around the final hash here to make it clear that it will be omitted if the aggregate key is either the taproot output key or the taproot internal key

Reply:
I've changed it to also say `or omitted`.